src/Raven.Client/Documents/Operations/Backups/RestoreBackupConfiguration.cs

```diff
@@ -34,6 +34,19 @@ public virtual DynamicJsonValue ToJson()
                 [nameof(Type)] = Type
             };
         }
+
+        internal virtual DynamicJsonValue ToAuditJson() =>
+            new()
+            {
+                [nameof(DatabaseName)] = DatabaseName,
+                [nameof(LastFileNameToRestore)] = LastFileNameToRestore,
+                [nameof(DataDirectory)] = DataDirectory,
+                [nameof(EncryptionKey)] = EncryptionKey,
```

Comment:
we need to avoid putting confidential information like this in the audit log

Reply:
Done.